### ISSUE TITLE
Not possible to install PyCryptodome under alternative package name ("Cryptodome") with Windows Precompiled

### ISSUE THREAD

**Issue by CodyMcCodeface** [Legrandin/pycryptodome#18]
I actually mentioned this already in [another recent ticket](https://github.com/Legrandin/pycryptodome/issues/16), but since this problem was not reflected in that ticket's title, and since it is not a good practice to report multiple issues in the same ticket in general, I'm hereby creating a new ticket for this issue alone. That being said, here goes:

It is currently not possible to install the Windows Precompiled version of PyCryptodome (i.e. using the wheel files) under the alternative package name ("Cryptodome"), but rather only under the "Crypto" package name which collides with PyCrypto.

For more details, please see the following StackOverflow question about this:

http://stackoverflow.com/questions/37504622/how-to-install-a-python-wheel-under-an-alternative-package-name-in-my-specific

It would therefore be super if you could provide wheel files also for installing under the "Cryptodome" package name.

(to make matters even worse, [your documentation](http://pycryptodome.readthedocs.io/en/latest/src/installation.html#windows-pre-compiled) falsely claims says that the Windows Precompiled version will install under the "Cryptodome" package name, which is what [my other recent ticket was about](https://github.com/Legrandin/pycryptodome/issues/16)).

**Comment by Varbin** [comment]
The name of the package for the Cryptodome namespace is "pycryptodome**x**" and for the Crypto namespace it is "pycryptodome". The documentation is about the "pycryptodome**x**" package. But  I agree that this should be noted somewhere (documentation and/or PyPi page).

**Comment by CodyMcCodeface** [comment]
Yes, I know that. The problem is that it is not possible to install it as a wheel with that command on Python 3.4 (see [the other ticket](https://github.com/Legrandin/pycryptodome/issues/16) for more details), and therefore I had to install the provided wheel file from the [PyPi page](https://pypi.python.org/pypi/pycryptodome) manually, like so:

> pip install c:\some\path\name-of-wheel-file.whl

(please see [my StackOverflow question about this](http://stackoverflow.com/questions/37504622/how-to-install-a-python-wheel-under-an-alternative-package-name-in-my-specific) for more details)

And those wheel files you provide on the PyPi page in turn seem to use only the "Crypto" package name, right?

Or is there a way to specify a separate package name even though you're **_manually_** installing a wheel file after all? In that case you can enter that as an accepted answer on my StackOverflow question as a bonus if you want. ;-)
